storages/backends/gcloud.py

```diff
@@ -153,7 +155,7 @@ def _save(self, name, content):
         content.name = cleaned_name
         encoded_name = self._encode_name(name)
         file = GoogleCloudFile(encoded_name, 'rw', self)
-        file.blob.upload_from_file(content, size=content.size)
+        file.blob.upload_from_file(content, size=content.size, content_type=file.mime_type)
```

Comment:
It's better to wrap lines at 79 columns (although the [Django Coding Style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/) allows up to 119 if needed for readability). Here you can put `content_type=...` on the next line without hurting readability, so please do.